styles/styles.css

```diff
@@ -0,0 +1,278 @@
+* {box-sizing: border-box;}
+
+button, input, select, textarea {
+  font-family: inherit;
+  font-size: 100%;
+}
+.skip-hidden {
+  position: absolute;
+  top: -10000px;
+  left: -10000px;
+  height: 1px;
+  width: 1px;
+  text-align: left;
+  overflow: hidden;
+}
+a.skip-hidden:focus {
+  left: 0;
+  top: 0;
+  width: auto;
+  height: auto;
+  overflow: visible;
+}
+header {
+  display: flex;
+  flex-direction: row;
+  background-image: url(header-image.jpeg);
+  background-repeat: no-repeat;
+  background-size: contain;
+  background-position: right;
+  background-color: #0F0C0C;
+  border: solid .1rem;
+  width: 85%;
+  min-width: 320px;
+  max-width: 1200px;
+  height: 25rem;
+}
+.logo-container {
+  margin-left: 5rem;
+  margin-top: 2.5rem;
+}
+h1 {
+  color: #32CD32;
+  font-size: 1.5rem;
+}
+.cart-price {
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap-reverse;
+  color: white;
+  margin-left: 33rem;
+  margin-top: 3rem;
+}
+.cart {
+  margin: .7rem;
+  align-self: flex-end;
+}
+.cart::after {
+  content: "$1234.56";
+  font-size: .8rem;
+}
+.cart-price button {
+  padding: .3rem;
+  color: white;
+  font-weight: bold;
+  background-color: #32CD32;
+  width: 7rem;
+  border-color: #32CD32;
+}
+.navigation {
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+  overflow: hidden;
+  background-color: #d3d3d3;
+  top: 0;
+  width: 85%;
+  min-width: 320px;
+  max-width: 1200px;
+}
+.nav {
+  float: left;
+  border-right: .1rem solid #bbb;
+  font-family: monospace;
+  font-size: 1.5rem;
+}
+.nav a {
+  display: block;
+  color: white;
+  text-align: center;
+  padding: 1rem 2rem;
+  text-decoration: none;
+}
+.nav a:hover {
+  background-color: #a9a9a9;
+}
+section {
+  margin-bottom: 8rem;
+  min-width: 320px;
+  max-width: 1200px;
```

Comment:
see above about max-widths. 

Also, you may need to center these with `margin: 0 auto;` so it's centered on wide screens